packages/react/test/specs/components/Popup/Popup-test.tsx

```diff
@@ -266,4 +263,44 @@ describe('Popup', () => {
       expect(contentElement.classList.contains(Popup.Content.className)).toEqual(true)
     })
   })
+
+  describe('keyboard event propagation', () => {
+    const expectPopupToStopPropagation = (
```

Comment:
Would rather to rename it to `expectPopupToHandleStopPropagation` as this tests both cases of when it should stop and not.

Reply:
will rename